go.mod

```diff
@@ -8,6 +8,8 @@ exclude k8s.io/kubernetes v1.13.0
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
+replace maistra.io/api v0.0.0-20221103173341-6ef6ed929778 => github.com/maistra/api v0.0.0-20230704084350-dfc96815fb16
```

Comment:
@jwendell @yxun do you know how to publish maistra.io/api to remove this hack?

Reply:
We have a vanity url set (e.g. https://github.com/maistra/maistra.github.io/blob/maistra-2.4/static/api) so I don't know why this is not working anymore. Perhaps we should rely on a tool like [vangen](https://github.com/leighmcculloch/vangen) to generate it instead?

Reply:
It would be also nice to have proper versions so we don't need to use those `v0.0.0-*` :)

Reply:
It's a shame... but I just forgot to run `go mod tidy` and we don't need to change the default branch.